### PR TITLE
Rework the main `Readme`

### PR DESCRIPTION
**Pull request by MGaetan89**
# Pull request

## Description

This PR updates the main Readme file.

## Changes made

- All badges are now from shields.io to have a consistent display.
- Provide an overview of Pillarbox's features.
- Add a section about the demo applications.
- Add a link to GitHub's documentation about adding a repository.
- Add links to each Pillarbox modules documentation.

## Checklist

- [x] Your branch has been rebased onto the `main` branch.
- [ ] APIs have been properly documented (if relevant).
- [x] The documentation has been updated (if relevant).
- [ ] New unit tests have been written (if relevant).
- [ ] The demo has been updated (if relevant).
- [x] All pull request status checks pass.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,63 +1,101 @@
...
+[![Last release](https://img.shields.io/github/v/release/SRGSSR/pillarbox-android?label=Release)](https://github.com/SRGSSR/pillarbox-android/releases)
+[![Android min SDK](https://img.shields.io/badge/Android-21%2B-34A853)](https://github.com/SRGSSR/pillarbox-android)
+[![Build status](https://img.shields.io/github/actions/workflow/status/SRGSSR/pillarbox-android/build.yml?label=Build)](https://github.com/SRGSSR/pillarbox-android/actions/workflows/build.yml)
+[![License](https://img.shields.io/github/license/SRGSSR/pillarbox-android?label=License)](https://github.com/SRGSSR/pillarbox-android/blob/main/LICENSE)
 
-## Compatibility
+Pillarbox is the modern SRG SSR multimedia player ecosystem, built on top of [AndroiX Media3](https://developer.android.com/media/media3). Pillarbox has been designed with robustness, flexibility, and  efficiency in mind, with full customization of:
+- Metadata and asset URL retrieval.
+- Asset resource loading, including support for Widevine and PlayReady.
+- Analytics and QoS integration.
```
Comment (StaehliJ) [inline docs/README.md:13]: QoS is currently not implemented, but in the road map.

File: docs/README.md [hunk 2]
```diff
@@ -1,63 +1,101 @@
...
+- User interface layout, in either [Compose](https://developer.android.com/develop/ui/compose/layouts) or [XML `View`s](https://developer.android.com/develop/ui/views/layout/declaring-layout).
```
Comment (StaehliJ) [inline docs/README.md:14]: PIllarbox is no ui, maybe specify that is the "surface", the bidule qui faire bouger les images".

File: docs/README.md [hunk 3]
```diff
@@ -1,63 +1,101 @@
...
 
-The library is suitable for applications running on android SDK 21 and above. The project is meant to be compiled with the latest Android version.
+Its robust player provides all essential playback features you might expect:
+- Audio and video (including 360° videos) playback.
+- Support for on-demand and live streams, with and without DVR.
```
Comment (StaehliJ) [inline docs/README.md:18]: Mention android MediaSession and Android auto integration.

File: docs/README.md [hunk 4]
```diff
@@ -1,63 +1,101 @@
...
+- Integration with the system playback user experience.
+- Playlist management, including bidirectional navigation.
+- Support for alternative audio tracks, audio description, subtitles, ...
+- Multiple instances support.
+- Picture-in-picture support.
```
Comment (StaehliJ) [inline docs/README.md:23]: Supported but PIllarbox provide nothing on this specific topic.

File: docs/README.md [hunk 5]
```diff
@@ -1,63 +1,101 @@
...
+```kotlin
+// Basic player features
+implementation("ch.srgssr.pillarbox:pillarbox-player:<pillarbox_version>")
 
-```gradle
-implementation("ch.srgssr.pillarbox:pillarbox-player:$LATEST_RELEASE_VERSION")
-// Library to handle SRG content through media urns
-implementation("ch.srgssr.pillarbox:pillarbox-core-business:$LATEST_RELEASE_VERSION")
+// Library to handle SRG SSR content through media urns
+implementation("ch.srgssr.pillarbox:pillarbox-core-business:<pillarbox_version>")
```
Comment (StaehliJ) [inline docs/README.md:71]: Maybe add the ui library too

```
// Library to display the video surface
implementation("ch.srgssr.pillarbox:pillarbox-ui:<pillarbox_version>")
```

**Review by StaehliJ — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 4 above)
Comment (MGaetan89) [reply docs/README.md:23]: That's also true for others points in this list (alternative tracks, on-demand/live support, ...)

Comment (StaehliJ) [reply docs/README.md:24]: Indeed, we can keep it then.

**Review by StaehliJ — APPROVED** [review]

**Review by Loic-Dumas — APPROVED** [review]

**Pull request merged**